lib/cluster.js

```diff
@@ -285,6 +287,8 @@ function masterInit() {
     var debugPort = process.debugPort + id;
     var hasDebugArg = false;
 
+    debugPort = debugPort % kDebugPortRange + kDebugPortStart;
```

Comment:
I missed it earlier today but there is a subtle change in behavior here that's also a potential bug.

Say debugPort equals 12345.  That's within the legal range but your commit changes it to 12345 % 65536 + 1024 = 13369.

Worse, with debugPort == 65000, it's changed to 65000 % 65536 + 1024 = 66024 and that's not a legal port number.